gcc/rust/rust-session-manager.cc

```diff
@@ -57,6 +58,8 @@
 #include "selftest.h"
 #include "tm.h"
 #include "rust-target.h"
+#include <sstream>
+#include <vector>
 
```

Comment:
These includes are prohibited here for obscure bootstrapping compiler reasons, but you can include `rust-system.h` instead